domain/build.gradle

```diff
@@ -44,7 +44,8 @@ dependencies {
       'androidx.lifecycle:lifecycle-livedata-ktx:2.2.0-alpha03',
       'com.google.code.gson:gson:2.8.6',
       'com.google.dagger:dagger:2.24',
-      "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+      "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version",
+      'androidx.exifinterface:exifinterface:1.0.0-rc01'
```

Comment:
What is this for?

Reply:
This is for getting the metadata of an uploaded image to decide whether it needs to be rotate before getting stored.